template.js

```diff
@@ -622,3 +652,47 @@ function determinateIsLoggingEnabled() {
 
   return data.logType === 'always';
 }
+
+function flattenPixelIdAndAccessTokenList(pixelIdAndAccessTokenArray) {
+  /*
```

Comment:
I think it's better to move this to README to keep tag smaller

Reply:
Sure. I added it for more context for you when reviewing the code.

Reply:
Removed this function entirely. It isn't needed anymore.